CHANGELOG.md

```diff
@@ -1,5 +1,69 @@
+* fix(endpoints): move AWS name map to JSv3, additional fixes for all-service endpoint compilation ([#614](https://github.com/awslabs/smithy-typescript/pull/614))
+* fix(endpoint): add default signing name ([#615](https://github.com/awslabs/smithy-typescript/pull/615))
+* fix(endpoint): deduplicate client context and built-in params with the same name ([#616](https://github.com/awslabs/smithy-typescript/pull/616))
+* fix(endpoint): do not modify context param names ([#617](https://github.com/awslabs/smithy-typescript/pull/617))
+* fix(endpoint): quote-wrap keys in generated ruleset ts file ([#618](https://github.com/awslabs/smithy-typescript/pull/618))
+* Use FileWriter from TypeScript Delegator to write EndpointV2 files  ([#619](https://github.com/awslabs/smithy-typescript/pull/619))
+* fix(types): expand custom endpoint type ([#622](https://github.com/awslabs/smithy-typescript/pull/622))
+* fix(codegen): limit the Bucket param dedupe to S3 service ([#626](https://github.com/awslabs/smithy-typescript/pull/626))
+* fix: trailing comma not allowed in ruleset ([#634](https://github.com/awslabs/smithy-typescript/pull/634))
+* Remove unconditional adding of util-base64 node and browser modules ([#636](https://github.com/awslabs/smithy-typescript/pull/636))
+* Fix the handling `intEnum` shapes when generating output value ([#643](https://github.com/awslabs/smithy-typescript/pull/643))
+* EndpointsV2: Do not error without aws.api#service trait ([#644](https://github.com/awslabs/smithy-typescript/pull/644))
+* Fix broken reference to `fail()` after jest-upgrade ([#645](https://github.com/awslabs/smithy-typescript/pull/645))
+* Validate required input query params ([#647](https://github.com/awslabs/smithy-typescript/pull/647),[#646](https://github.com/awslabs/smithy-typescript/pull/646))
+* fix(codegen): make util retry conditional ([#651](https://github.com/awslabs/smithy-typescript/pull/651))
+* chore: remove aws trait dependency ([#652](https://github.com/awslabs/smithy-typescript/pull/652))
+* fix: add idempotencyToken generation if member is queryParam ([#655](https://github.com/awslabs/smithy-typescript/pull/655))
+* Fix Error printout for protocol-response tests ([#657](https://github.com/awslabs/smithy-typescript/pull/657))
+* Read staticContextParams from operation ([#658](https://github.com/awslabs/smithy-typescript/pull/658))
+* Fix codegen windows ([#661](https://github.com/awslabs/smithy-typescript/pull/661))
+* Fix missing underscore prefix in DefaultsMode Provider ([#671](https://github.com/awslabs/smithy-typescript/pull/671))
+
+### Miscellaneous
+
+* Update versions in README example ([#601](https://github.com/awslabs/smithy-typescript/pull/601))
+* Update links to point to smithy.io ([#637](https://github.com/awslabs/smithy-typescript/pull/637))
+* Add smithy as codeowners for the whole repo ([#664](https://github.com/awslabs/smithy-typescript/pull/664))
```

Comment:
These are internal repo setup. They don't need to go into changelog. Only keep stuff that would be relevant to consumers of this package. Look through all the PRs in this change.